docs/Storm-Scheduler.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Scheduler
+layout: documentation
+documentation: true
+---
+
+Storm now has 4 kinds of build-in schedulers: [DefaultScheduler](https://github.com/apache/storm/blob/1.x-branch/storm-core/src/clj/org/apache/storm/scheduler/DefaultScheduler.clj), [IsolationScheduler](https://github.com/apache/storm/blob/1.x-branch/storm-core/src/clj/org/apache/storm/scheduler/IsolationScheduler.clj), [MultitenantScheduler](https://github.com/apache/storm/blob/1.x-branch/storm-core/src/jvm/org/apache/storm/scheduler/multitenant/MultitenantScheduler.java), [ResourceAwareScheduler](Resource_Aware_Scheduler_overview.html). 
```

Comment:
This is linked to storm site Resource_Aware_Scheduler_overview**.html**. It will be good after this PR merged and published.